frontend/src/components/customer/ProfileDetailsCard.tsx/ProfileDetailsCard.tsx

```diff
@@ -0,0 +1,480 @@
+            reader.onload = (e) => {
+                setProfilePicture(e.target?.result as string);
+            };
+            reader.readAsDataURL(file);
+        }
+    };
+
+    const defaultProfilePicture = ProfilePic;
+
+    const handleDeletePicture = () => {
+        setProfilePicture(defaultProfilePicture);
+    };
+
+    const handleCancel = () => {
+        form.reset();
+        setValue(null);
+        setError(null);
+    };
+
+    return (
+        <form onSubmit={handleSubmit(onSubmit)}>
+            <Paper
+                elevation={2}
+                sx={{
+                    mt: "3rem",
+                    mb: "3rem",
+                }}
+            >
+                <Grid container direction="column">
+                    <Grid item sx={{ margin: "2rem" }}>
+                        <Heading
+                            variant="h2"
+                            color="secondary"
+                            style={{ marginLeft: "1rem" }}
+                        >
+                            Profile Details
+                        </Heading>
+                    </Grid>
+                    <Grid
+                        item
+                        sx={{
+                            ml: "2rem",
+                            mr: "2rem",
+                        }}
+                    >
+                        <Grid
+                            container
+                            direction="row"
+                            alignContent="flex-start"
+                            justifyContent="center"
+                            alignItems="center"
+                            spacing={3}
+                        >
+                            <Grid item xs={4}>
+                                <Grid container direction="column" gap={2}>
+                                    <Grid item>
+                                        <Avatar
+                                            src={profilePicture}
+                                            sx={{
+                                                width: "11rem",
+                                                height: "11rem",
+                                            }}
+                                        />
+                                    </Grid>
+                                    <Grid container direction="row" spacing={2}>
+                                        <Grid item>
+                                            <input
+                                                type="file"
+                                                accept="image/*"
+                                                id="profile-picture-upload"
+                                                style={{ display: "none" }}
+                                                onChange={handleFileUpload}
+                                            />
+                                            <label htmlFor="profile-picture-upload">
```

Comment:
Don't create an upload button, use the Avatar as an image input directly. Might need to use external libraries to make the Avatar can serve both as current pfp display and input as well